src/Sentry/Laravel/Tracing/Integrations/LighthouseIntegration.php

```diff
@@ -80,7 +80,7 @@ public function setupOnce(): void
     private function processEvent(Event $event, Options $options): void
     {
         // Detect if we are processing a GraphQL request, if not skip processing the event
-        if (!Str::startsWith($event->getTransaction(), 'lighthouse?')) {
+        if (!Str::startsWith($event->getTransaction() ?? '', 'lighthouse?')) {
```

Comment:
How about we rewrite this as:

```suggestion
        if ($event->getTransaction() === null || !Str::startsWith($event->getTransaction(), 'lighthouse?')) {
```

That reads better to me and possibly saves some string operations that are not needed anyway.